src/test/java/core/basesyntax/service/impl/ReadFromFileImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReadFromFile;
+import java.util.List;
+import java.util.stream.IntStream;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReadFromFileImplTest {
+    private ReadFromFile reader;
+
+    @Before
+    public void setUp() {
+        reader = new ReadFromFileImpl();
+    }
+
+    @Test
+    public void readFromFile_wrongPath_notOk() {
+        String wrongPath = "src/test/java/resources/file123.txt";
+        try {
+            reader.readFormFile(wrongPath);
+        } catch (RuntimeException e) {
+            assertEquals("No such file found!", e.getMessage());
+        }
+    }
+
+    @Test
+    public void readFromFile_nullPath_notOk() {
+        String nullPath = null;
+        try {
+            reader.readFormFile(nullPath);
+        } catch (RuntimeException e) {
+            assertEquals("No path has been put", e.getMessage());
+        }
+    }
```

Comment:
what if exception will not thrown, should test pass in this case? 
now looks like it will pass